test/dotnet-publish.Tests/PublishTests.cs

```diff
@@ -221,16 +215,16 @@ public void TestLibraryBindingRedirectGeneration()
             var publishCommand = new PublishCommand(lesserTestProject, "net451");
             publishCommand.Execute().Should().Pass();
 
-            publishCommand.GetOutputDirectory().Should().HaveFile("TestLibraryLesser.dll");
+            publishCommand.GetOutputDirectory().Should().HaveFile("TestLibraryLesser.exe");
```

Comment:
This was testing binding redirects. Now that '--infer-runtimes' is removed publish cannot work on libraries which do not have any RIDs.